Plugins~/Src/MeshSyncClient/Include/MeshSyncClient/msInstancesManager.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "MeshSync/MeshSync.h"
+#include "MeshSync/SceneGraph/msIdentifier.h"
+#include<map>
+#include <unordered_map>
+#include "msTransformManager.h"
+
+#ifndef msRuntime
+
+msDeclClassPtr(InstanceInfo)
+
+namespace ms {
+
+    class InstancesManager : public TransformManager
```

Comment:
Can you add some documentation for this class?

Reply:
Added XML comments on class and functions